configuration/configmgr/configmgrlib/SchemaValue.cpp

```diff
@@ -59,6 +60,8 @@ SchemaValue::SchemaValue(const SchemaValue &value)
     // special processing? Maybe after inserting?
     std::vector<std::shared_ptr<SchemaValue>> m_mirrorToSchemaValues;
     std::vector<std::weak_ptr<SchemaValue>> m_pUniqueValueSetRefs;    // this value serves as the key from which values are valid
+
+    m_ordinal = 0;  // assigned when inserted
```

Comment:
constructor initializer list?

Reply:
The constructor for a SchemaValue will set it to 0, however, should the code in the copy constructor, where this line is, accidentally copy the value from the source object, this will make sure it is 0 when the copy is complete. I felt it just safer to explicitly set it to 0 before exit.